simulate.py

```diff
@@ -97,7 +162,8 @@ def hamming(seq1,seq2):
 	flip1 = random.randint(0,1)
 	flip2 = random.randint(0,1)
 	if flip1:
-		seq1 = rc(seq1)
+		if gen == 0:
+			seq1 = rc(seq1)
```

Comment:
I think we should randomly flip the first genome no matter what. We don't gain anything by ensuring that simulated genome_0 is identical to input genome, do we?